<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMH::Internal::PmhViewer</class>
 <widget class="QWidget" name="PMH::Internal::PmhViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="0">
    <widget class="QGroupBox" name="simpleBox">
     <property name="title">
      <string>ICD 10</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="1" column="0" colspan="2">
       <widget class="QListView" name="icdCodes">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="simple_icd10">
        <property name="text">
         <string>Manage &amp;ICD10 Codes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Label and date</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&amp;Label:</string>
        </property>
        <property name="buddy">
         <cstring>personalLabel</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="personalLabel"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&amp;Date:</string>
        </property>
        <property name="buddy">
         <cstring>simple_date</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="simple_date">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="makePrivateBox">
        <property name="text">
         <string>This PMHx is &amp;private</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>General information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Status:</string>
        </property>
        <property name="buddy">
         <cstring>statusCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Type:</string>
        </property>
        <property name="buddy">
         <cstring>typeCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QComboBox" name="statusCombo"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QComboBox" name="typeCombo"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>unsure</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QSlider" name="confIndexSlider">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>proven</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&amp;Confidence index:</string>
        </property>
        <property name="buddy">
         <cstring>confIndexSlider</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Category</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QTreeView" name="categoryTreeView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="episodesTab">
      <attribute name="title">
       <string>Episodes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="PMH::PmhEpisodeViewer" name="episodeViewer" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="managementTab">
      <attribute name="title">
       <string>Management</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="contactTab">
      <attribute name="title">
       <string>Contacts</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="linkTab">
      <attribute name="title">
       <string>Links</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabComment">
      <attribute name="title">
       <string>Comment</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="Editor::TextEditor" name="comment"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">Past Medical History for PATIENT NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Editor::TextEditor</class>
   <extends>QTextEdit</extends>
   <header location="global">texteditorplugin/texteditor.h</header>
  </customwidget>
  <customwidget>
   <class>PMH::PmhEpisodeViewer</class>
   <extends>QWidget</extends>
   <header location="global">pmhplugin/pmhepisodeviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>personalLabel</tabstop>
  <tabstop>simple_date</tabstop>
  <tabstop>makePrivateBox</tabstop>
  <tabstop>typeCombo</tabstop>
  <tabstop>statusCombo</tabstop>
  <tabstop>confIndexSlider</tabstop>
  <tabstop>simple_icd10</tabstop>
  <tabstop>icdCodes</tabstop>
  <tabstop>categoryTreeView</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>comment</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
